files/mods.json

```diff
@@ -371,7 +371,7 @@
         "creator": "Sk1er LLC",
         "discordcode": "sk1er",
         "command": "/patcher",
-        "description": "Patcher optimizes the game in every possible way, fixes 70+ vanilla bugs along with some QOL features while maintaining both compatibility, and has almost no tradeoffs.",
+        "description": "Optimizes the game and fixes many bugs, with no tradeoffs",
```

Comment:
the qol aspect of patcher is very important, would be nice to see it mentioned

Reply:
give examples of its qol aspect

Reply:
chat timestamps (sbe users love this for some reason), compact chat, left hand for left handed people (kinda a stretch here but)